modules/fts/pages/fts-architecture.adoc

```diff
@@ -0,0 +1,54 @@
+= FTS Architecture
+
+
+[abstract]
+
+
+The FTS Architecture is explained in the form of diagrams. Each diagram is co-located with its textual description.
+
+== Search Service Architecture
+
+The _Search_ Service provides extensive capabilities for natural-language querying.
+These include:
+
+* _Language-aware_ searching; allowing users to search for, say, the word `beauties`, and additionally obtain results for `beauty` and `beautiful`.
+* _Scoring_ of results, according to relevancy; allowing users to obtain result-sets that only contain documents awarded the highest scores.
+This keeps result-sets manageably small, even when the total number of documents returned is extremely large.
+* _Fast indexes_, which support a wide range of possible text-searches.
+
+The indexes that the Search Service creates and uses are entirely separate from and different to those of the Index Service.
+
+However, if the Index and Query Services are also deployed on the cluster, the facilities of the Search Service can be called on by means of the _N1QL_ language &#8212; which provides the principal interface to the Query and Index Services.
+See xref:learn:services-and-indexes/services/search-service.adoc#search_via_query[Accessing the Search Service with N1QL], below.
+
+The Search Service depends on the Data Service, as follows:
+
+[#search_service_architecture]
+image::services-and-indexes/services/searchServiceArchitecture2.png[,500,align=left]
+
+The shows the following:
+
+* The _Data Service_ uses the DCP protocol to stream data-mutations as batches, from the producer to multiple consumers (_vBuckets_), instantiated across the search nodes.
+
+* When a _search index_ is created by means of the _Search Service_, search-index data-handling for the vBuckets is divided equally among the established search-index partitions.
+
+For example, if the number of vBuckets is 120, and the number of search-index partitions chosen is 6, each search-index partition holds data for 20 vbuckets.
```

Comment:
This may technically be true, but I'm not sure it's important enough to highlight here.  In practice there are always 1024 vBuckets, and we now default to 1 search partition, so while this example is not wrong, it is mostly theoretical.

Reply:
I'm basing this comment on that fact that the title of the page includes "introduction", and this level of detail isn't needed appropriate in an introduction.

Reply:
Update to: For example, when the number of vBuckets is 1024, and the number of search-index partitions chosen is 6, each search-index partition holds data for ~171 vBuckets.